### ISSUE TITLE
flake8-logging-format catches more things than ruff

### ISSUE THREAD

**Issue by alexprengere** [charliermarsh/ruff#2273]
Tested with ruff `0.0.236` and the latest flake8-logging-format. This is an error (G004) with flake8 but passes with ruff:

```python
import logging
log = logging.getLogger(__name__)
msg = "message"
log.warning(f"A {msg}")
```

When using directly `logging.warning`, both fail.
Using `logging.getLogger` is a common idiom when logging messages on large projects, so it would be super useful to catch it as well.

**Comment by edgarrmondragon** [comment]
`flake8-logging-format` only excludes the `warnings.` and `parser.` call paths, so `log.warning` is identified as a logger call. However, `ruff` only _includes_ `logging` and `*logger` call paths, so `log.warning` is not identified as a logger.

This is probably easy to fix by changing `is_logger_candidate`: https://github.com/charliermarsh/ruff/blob/3ec46f093604c4fe79f724a5f927b241387a239c/src/ast/helpers.rs#L1009-L1019

**Comment by charliermarsh** [comment]
Yeah we could extend to anything that includes log, I think.

**Comment by charliermarsh** [comment]
(It’s also possible that I was wrong to restrict the checks in this way.)

**Comment by Flowake** [comment]
The issue of correctly identifying how loggers are named arises for multiple rules.
Maybe we could define in one place the accepted names for some object to be considered a logger ? (I'm thinking about TRY400 which is only triggered by the logger being called "logging" or "logger")

**Comment by charliermarsh** [comment]
We do have that centralized helper (`is_logger_candidate`), which TRY400 uses too. So if we just tweak that, it'll fix this for both rules.

**Comment by alexprengere** [comment]
Thanks a lot for the quick fix!
